packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -97,7 +99,7 @@ export class GraphQLPersonalizeService {
       return data?.layout?.item
         ? {
             // CDP expects content id format `<id>_<language>_<version>` (lowercase)
-            contentId: `${data.layout.item.id}_${language}_${data.layout.item.version}`.toLowerCase(),
+            contentId: `${EMBEDDED}${data.layout.item.id}_${language}`.toLowerCase(),
```

Comment:
You need to reflect same changes for `personalize-middleware` tests - https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.test.ts